create_toc.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+"""Create table of contents with all epics."""
+
+import os
+
+
+def get_toc():
+    """Get table of contents with all epics."""
+    epic_dirs = [d for d in os.listdir(".") if os.path.isdir(d) and d[:1].isdigit()]
+    toc = []
+    for epic_dir in epic_dirs:
+        try:
+            num = int(epic_dir.split("-", 1)[0])
+            if not 0 <= num < 1000:
+                raise ValueError
+        except ValueError:
+            raise RuntimeError(f"Invalid directory name {epic_dir}")
+        spec_path = os.path.join(epic_dir, "technical_specification.md")
+        if not os.path.exists(spec_path):
+            raise RecursionError(f"No technical_specification.md found in {epic_dir}")
+        for line in open(spec_path):
+            if line.startswith("#"):
+                title = line[1:].strip()
+                break
+        else:
+            title = None
+        if not title:
+            raise RuntimeError(f"No title found in {spec_path}")
+        try:
+            desc, name = title.rsplit("(", 1)
+            desc = desc.rstrip()
+            name = name[:-1].strip().capitalize()
```

Comment:
```suggestion
            name = name[:-1].strip().title()
```
I think title case looks a little nicer with multi-word titles (Wood Ant vs Wood ant). It's a style choice so take it or leave it.